### PR TITLE
Update sharepoint-file-access-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the use-app-enforced-restrictions-in-Sharepoint as the conditional access policies are automatically created. when you implement.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8065

File: microsoft-365/security/office-365-security/sharepoint-file-access-policies.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The following table lists the policies you either need to review and update or c
 
 ## Use app-enforced restrictions in SharePoint
 
-If you implement access controls in SharePoint, you must create this Conditional Access policy in Azure AD to tell Azure AD to enforce the policies you configure in SharePoint. This policy applies to all users, but only affects access to the sites you specify using PowerShell when you create the access controls in SharePoint.
+If you implement access controls in SharePoint, Conditional Access policies are created in Azure AD to tell Azure AD to enforce the policies you configure in SharePoint. By default, this policy applies to all users, but only affects access to the sites you specify using PowerShell when you create the access controls in SharePoint. The policy can also scoped for specific users, groups or sites.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/sharepoint-file-access-policies.md]:
```suggestion
If you implement access controls in SharePoint, Conditional Access policies are created in Azure AD to tell Azure AD to enforce the policies you configure in SharePoint. By default, this policy applies to all users, but only affects access to the sites you specify using PowerShell when you create the access controls in SharePoint. The policy can also be scoped for specific users, groups, or sites.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**